### DISCUSSION TITLE
Adopting two plugins

### DISCUSSION THREAD

**Discussion by yasn77** [mise-plugins/.github#2] [Transfer Request]
Hi,

I currently have two plugins that I would like adopted as official plugins:

- https://github.com/yasn77/mise-jless
- https://github.com/yasn77/asdf-jira-cli

Not sure what the next steps would be, but happy to try and update the repos if needed.

**Comment by jdx** [comment]
if you start a transfer to my personal account (jdx) I'll add them to mise-plugins.

**Reply by yasn77** [reply]
Done... Thanks
